assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -137,7 +137,8 @@ export default Component.extend({
 
     if (
       this.chatChannel &&
-      this.registeredChatChannelId !== this.chatChannel?.id
```

Comment:
There was no need for safe operation since nothing else would be evaluated if `this.chatChannel` was null or undefined